core/migrations/0151_update_dpia_needed_model.py

```diff
@@ -0,0 +1,46 @@
+# Generated by Django 4.2.8 on 2024-01-11 09:27
+
+from django.db import migrations
+
+
+REFERENCE_DATA_MODEL = "DataProcess"
+
+OPTIONS = (
+    "large_scale",
+    "matching_data",
+    "children",
+    "innovative_technology",
+    "no_privacy_information",
+    "tracking",
+    "monitor_public_place",
+    "none_of_the_above",
+    "dont_know",
+)
+
+
+def update_reference_data(apps, schema_editor):
+    SpecificReferenceDataModel = apps.get_model("core", REFERENCE_DATA_MODEL)
+
+    SpecificReferenceDataModel.objects.get(code="special_category").delete()
+    SpecificReferenceDataModel.objects.get(code="endanger_safety").delete()
+
+    SpecificReferenceDataModel.objects.create(
+        code="monitor_public_place",
+        name="Systematically monitor a pubicaly-accessible place",
```

Comment:
Sorry I spelt it wrong in the issue.
```suggestion
        name="Systematically monitor a publicly-accessible place",
```